src/main/scala/org/apache/spark/sql/hive/llap/LlapContext.scala

```diff
@@ -124,6 +122,44 @@ class LlapContext(sc: SparkContext,
       Seq.empty
     }
   }
+
+  override protected[sql] def executePlan(plan: LogicalPlan): super.QueryExecution = {
+    new this.QueryExecution(plan)
+    super.executePlan(plan)
+  }
+
+  private[sql] class QueryExecution(logicalPlan: LogicalPlan)
+    extends super.QueryExecution(logicalPlan)  {
+
+    /**
+     * Access control for Desc Table Function
+     */
+    val checkDesc = executedPlan match {
```

Comment:
+1 for removing this.
Generally, that's not a good idea to consider someone else in the future.
Most people (like me) will be confused here and will waste their time in order try to find where this variable used.